experiments/gail_benchmark.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# This script finds the the mean and standard error of episode return after
+# training GAIL on benchmark tasks.
+#
+# The CSV can be modified and regenerated from following spreadsheet:
+# https://docs.google.com/spreadsheets/d/1MZYTr23ddwhO2PrNI2EalKleoULXlsH5y9oqawt22pY/edit?usp=sharing
+#
+# When regenerating csv, remember that each line needs to end with ',' to
+# work properly.
+#
+# NOTE: Since I don't have walker experts available right now, I also
+# deleted the walker entries from the csv. TODO(shwang): Add them back in
+# by regenerating the csv after these experts are available.
+
+if $(command -v gdate > /dev/null); then
+  DATE_CMD=gdate  # macOS compatibility
+else
+  DATE_CMD=date
+fi
+
+TIMESTAMP=$(${DATE_CMD} --iso-8601=seconds)
+OUTPUT_DIR="output/gail_benchmark/${TIMESTAMP}"
+mkdir -p "${OUTPUT_DIR}"
+
+SEEDS="0 1 2"
+
+parallel -j 25% --header : --results ${OUTPUT_DIR}/parallel/ --colsep , --progress \
+  python -m imitation.scripts.train \
+  with \
+  fast \
```

Comment:
Do we want `fast`? We won't expect to reproduce GAIL results with this, right? Perhaps change to `with "$@"` so the bash script passes through command line arguments, letting you run it with `fast` if you want.

Reply:
My bad, I thought I had pushed a change that removed this to this. `"$@"` sounds like a great idea

Reply:
Looks like this isn't sufficient to have fast runs, because the `n_epochs=50000` overrides the named config! I'll probably add a `--fast` flag later when doing #62.

Reply:
Ah that is annoying! You could relocate the `"$@"` to the end, and then the user could override the previous `n_gen_steps_per_epoch` manually: `fast n_gen_steps_per_epoch=1`. But that's quite hacky. `--fast` flag is probably better.